src/Symfony/Component/Workflow/Exception/InvalidArgumentException.php

```diff
@@ -11,10 +11,25 @@
 
 namespace Symfony\Component\Workflow\Exception;
 
+use Symfony\Component\Workflow\WorkflowInterface;
+
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  * @author Grégoire Pineau <lyrixx@lyrixx.info>
  */
 class InvalidArgumentException extends \InvalidArgumentException implements ExceptionInterface
 {
+    public static function tooManyWorkflowsMatching(object $subject, array $workflows, string $message = '')
```

Comment:
Not sure we can use the `object` type declaration here 🤔

Reply:
Indeed :)